java/bundles/org.eclipse.set.core/src/org/eclipse/set/core/configurationservice/UserConfigurationServiceImpl.java

```diff
@@ -157,6 +161,7 @@ public void addKnownVersions(final String version) throws IOException {
 	@Override
 	public void setLastFileOpenPath(final Path path) {
 		configuration.lastFileOpenPath = path;
+		getLastOpenFiles().addFirst(path);
```

Comment:
@TruongQuangSB We should also avoid duplicates.

```suggestion
		List<Path> lastOpenFiles = getLastOpenFiles();
		if (lastOpenFiles.contains(path)) {
			// remove path if already inside open files to avoid duplicates
			lastOpenFiles.remove(path);
		}
		lastOpenFiles.addFirst(path);
```

Reply:
here should be compare path als string